templates/README.md.gotmpl

```diff
@@ -198,12 +198,36 @@ In general, PRs are welcome. We follow the typical "fork-and-pull" Git workflow.
 
  1. **Fork** the repo on GitHub
  2. **Clone** the project to your own machine
- 3. **Commit** changes to your own branch
- 4. **Push** your work back up to your fork
- 5. Submit a **Pull Request** so that we can review your changes
+ 3. [**Setup pre-commit**](####) to verify code style
+ 4. **Commit** changes to your own branch
+ 5. **Push** your work back up to your fork
+ 6. Submit a **Pull Request** so that we can review your changes
 
 **NOTE:** Be sure to merge the latest changes from "upstream" before making a pull request!
 
+#### Setup
+
+Before commit please setup [pre-commit](https://pre-commit.com/) following
+
+##### MacOs
+  ```
+  brew install pre-commit
+  pre-commit install --install-hooks
+  ```
+
+##### Linux
+  ```
+  pip install pre-commit
+  pre-commit install --install-hooks
+  ```
+
+##### Windows
+  ```
+  pip install pre-commit
+  pre-commit install --install-hooks
+  ```
+
```

Comment:
And a section only when `test/src/*` files are present:

### Testing

This module supports automated tests using `terratest`. There are a few ways to run them. 

#### Running Tests in Docker

For docker tests, we assume you have `docker` already installed. Then export your AWS credentials (e.g. `AWS_SECRET_KEY` and `AWS_ACCESS_KEY`) before running the following command:
 
```
make -C test/src docker/test
```

### Running Natively

The following command will run `terratests` locally on your workstation, but assumes that you have working go 1.12+ environment already setup. If you don't have that, consider using the Docker testing. Make sure you have exported your credentials to the environment (E.g `AWS_SECRET_KEY`, AWS_ACCESS_KEY_ID`) before running the commands.

`make -C test/src all`